src/engraving/internal/engravingconfiguration.h

```diff
@@ -38,6 +39,7 @@ class EngravingConfiguration : public IEngravingConfiguration, public async::Asy
     INJECT(engraving, mu::framework::IGlobalConfiguration, globalConfiguration)
     INJECT(engraving, mu::ui::IUiConfiguration, uiConfiguration)
     INJECT(engraving, mu::accessibility::IAccessibilityConfiguration, accessibilityConfiguration)
+    INJECT(engraving, notation::INotationConfiguration, notationConfiguration)
```

Comment:
This is not a good solution. Now we have a circular dependency between the engraving and notation modules

Reply:
I guess I hadn't fully considered that. I'll close this PR for now